pontoon/base/templatetags/helpers.py

```diff
@@ -268,7 +268,7 @@ def _serialize_elements(elements):
             elif type(element.expression) == ast.MessageReference:
                 response += '{ ' + element.expression.id.name + ' }'
 
-            elif hasattr(element, 'expression') and hasattr(element.expression, 'variants'):
+            elif hasattr(element.expression, 'variants'):
```

Comment:
I think all these ifs should use `isinstance`. You wont' need the `hasattr` check then at all.

Reply:
So in this particular I could use this:
```python
elif isinstance(element.expression, ast.SelectExpression):
```

If you agree with the proposal in https://github.com/mozilla/pontoon/pull/840#discussion_r166584643, I'd rather keep it in line with the JS code.